pyquil/tests/test_examples.py

```diff
@@ -0,0 +1,54 @@
+
+import subprocess
+from subprocess import Popen, PIPE, STDOUT
+
+
+# As bare minimum test that all example scripts run without error
+
+def test_meyer_penny_game():
```

Comment:
until we figure out how to spin up a qvm for semaphore, I think you need to mark several/most/all of these tests with the `forest` fixture which skips a test if it can't connect to a QVM

Reply:
xref https://github.com/rigetticomputing/pyquil/issues/424

Reply:
Will do.